CefSharp/CefCustomScheme.cs

```diff
@@ -119,13 +119,12 @@ public CefCustomScheme()
         /// <returns>list of scheme objects</returns>
         public static List<CefCustomScheme> ParseCommandLineArguments(IEnumerable<string> args)
         {
-            var schemes = args.FirstOrDefault(a => a.StartsWith(CefSharpArguments.CustomSchemeArgument));
+            var schemes = args.GetArgumentValue(CefSharpArguments.CustomSchemeArgument);
```

Comment:
Have you tested these changes?

Reply:
Yes... and I even spotted an existing "bug". Some of the parameters sent to the process are duplicated, that's why the FirstOrDefault works but not not SIngleOrDefault.

Reply:
Which paramaters are duplicated?

Reply:
Those defined in CefSharpArguments

Reply:
Interesting, thanks